app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -839,6 +839,8 @@ fun MessageList(
     selectedMessageId: String?
 ) {
     val prevItemCount = remember { mutableStateOf(lazyPagingMessages.itemCount) }
+    val readLastMessageAtStartTriggered = remember { mutableStateOf(false) }
```

Comment:
This state is not being set to `true` anywhere 🤔

Reply:
Wow, moving function I forgot about that...